kong/tools/faker.lua

```diff
@@ -36,6 +36,8 @@ Faker.FIXTURES = {
     { name = "API TESTS 8 (dns)", public_dns = "dns1.com", target_url = "http://127.0.0.1:7771" },
     { name = "API TESTS 9 (dns)", public_dns = "dns2.com", target_url = "http://localhost:7771" },
 
+    { name = "API TESTS 10 (ssl)", public_dns = "localhost", target_url = "http://mockbin.com" },
```

Comment:
`Faker.FIXTURES` doesn't exist anymore. Each test suite only create what it needs, and with more descriptive names than those fixtures had.

See `spec_helper.insert_fixtures` and its usage in the current plugins tests